### PR TITLE
[#147419311] Add feedback form

### PR DESCRIPTION
**Pull request by RamilyaNigmatullina**

### CODE REVIEW

File: app/controllers/feedbacks_controller.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
+class FeedbacksController < ApplicationController
+  expose(:feedback)
+
+  def new
+  end
+
+  def create
+    ApplicationMailer.feedback(feedback).deliver_now! if feedback.save
```
Comment (ShaidullinArtur) [inline app/controllers/feedbacks_controller.rb]: I think we need to use `deliver_later`

File: app/views/feedbacks/new.html.slim [hunk 2]
```diff
@@ -0,0 +1,26 @@
+ruby:
+  title("Contact us")
+  description("Contact the team")
+  keywords("Contact, Email, Touch, Getting in touch")
+- meta robots: "noindex"
```
Comment (ShaidullinArtur) [inline app/views/feedbacks/new.html.slim]: I think we can move this line to `ruby:` block

**Review by ShaidullinArtur — APPROVED** [review]
2 comments and 👍

File: app/controllers/feedbacks_controller.rb (hunk 1 above)
Comment (RamilyaNigmatullina) [reply app/controllers/feedbacks_controller.rb]: For using `deliver_later` feedback should have id parameter. Should I add module `GlobalID::Identification`and add id and than find then object by id like this?

```
class Feedback
  include GlobalID::Identification
  
  def id 
    ...   
  end    

  def self.find(id)
    ...   
  end 
end

**Pull request merged**